website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -282,7 +283,7 @@ def model(dbt, session):
 
 </WHCode>
 
-**Note:** Incremental models aren't currently supported on BigQuery.
+**Note:** Incremental models are supported on BigQuery/Dataproc for the `merge` incremental strategy. The `insert_overwrite` strategy is not yet supported.
```

Comment:
@jtcohen6 thank you for coming to office hours for dbt Python models yesterday and noticing this key detail!